airbyte-webapp/src/components/connectorBuilder/Builder/BuilderField.tsx

```diff
@@ -123,12 +123,14 @@ const InnerBuilderField: React.FC<BuilderFieldProps & FastFieldProps<unknown>> =
         />
       )}
       {props.type === "array" && (
-        <ArrayField
-          name={path}
-          value={(field.value as string[] | undefined) ?? []}
-          setValue={setValue}
-          error={hasError}
-        />
+        <div data-testid={`tag-input-${path}`}>
```

Comment:
Just a note - I have a PR up here that sets a testid on the actual input: https://github.com/airbytehq/airbyte-platform/pull/50

Can be consumed in a follow-up PR though.